app/components/app_patient_table_component.rb

```diff
@@ -0,0 +1,48 @@
+class AppPatientTableComponent < ViewComponent::Base
+  def call
```

Comment:
This is another way of defining ViewComponent templates, which I think makes most sense in this context given that it's all Ruby and no HTML currently. https://viewcomponent.org/guide/templates.html#call